### ISSUE TITLE
Understanding Gradient Descent in Neural Network · Rongjia Liu

### ISSUE THREAD

**Issue by jackliu234** [jackliu234/jackliu234.github.io#63]
https://jackliu234.com/2019/08/gradient-descent/ 

 Understanding Gradient Descent in Neural Network - Rongjia Liu

**Comment by allenfrostline** [comment]
啊！你网站界面更新了啊
